fs.go

```diff
@@ -4,82 +4,151 @@ import (
 	"errors"
 	"io"
 	"os"
+	"time"
 )
 
 var (
-	ErrClosed       = errors.New("file: Writing on closed file.")
-	ErrReadOnly     = errors.New("this is a read-only filesystem")
-	ErrNotSupported = errors.New("feature not supported")
+	ErrReadOnly        = errors.New("read-only filesystem")
```

Comment:
yep, and no. I need to fit the documentation and this means, use os.PathError and os.LinkError in all the function to mimic the standard library